### ISSUE TITLE
Anyway to get the file name?

### ISSUE THREAD

**Issue by MohammedAlasaad** [UgnisSoftware/react-spreadsheet-import#143]
Hi
Is there anyway to get the filename when user upload file and submit it from any step?
Thx

**Comment by je-kr** [comment]
Hello !
Did you find a solution to this ?
Thank you !

**Comment by masiulis** [comment]
Sorry for a late reply, this is addressed in #168 and will added in the next release.

```jsx
 <ReactSpreadsheetImport
        onSubmit={(data, file) => {
          console.log(file.name)
        }}/>
```
